scripts/globals/mobskills/guided_missile.lua

```diff
@@ -24,7 +24,7 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local ftp300 = 3
     local numhits = math.random(1, 3)
     local accmod = 1
-    local info = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, 1, xi.mobskills.magicalTpBonus.NO_EFFECT, numhits * ftp100, numhits * ftp200, numhits * ftp300)
```

Comment:
This should go into the module

Reply:
> All these changes should go into the module

It is the same change in both the module and the normal file, should I leave the normal file broken?

Reply:
Or I can change the file back to LSB version

Reply:
Yep, I removed that change!